script/storybook-deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+if [[ $( git diff --name-only HEAD~1..HEAD webpack/ package.json | wc -l ) -ne 0 ]]; then
+  REPO_SLUG_ARRAY=(${TRAVIS_REPO_SLUG//\// })
+  REPO_OWNER=${REPO_SLUG_ARRAY[0]}
+  REPO_NAME=${REPO_SLUG_ARRAY[1]}
+  DEPLOY_PATH=storybook-static
+
+
+  DEPLOY_SUBDOMAIN_UNFORMATTED_LIST=()
+  if [ "$TRAVIS_PULL_REQUEST" != "false" ]
+  then
+    npm run build-storybook -c storybook -o .out
+    DEPLOY_SUBDOMAIN_UNFORMATTED_LIST+=(${TRAVIS_PULL_REQUEST}-pr)
+  fi
+
+
+  for DEPLOY_SUBDOMAIN_UNFORMATTED in "${DEPLOY_SUBDOMAIN_UNFORMATTED_LIST[@]}"
```

Comment:
Why don't you nest this inside the `if` statement? You can even omit the whole array in favor of a single variable.